Kinvey.Core/Auth/User.cs

```diff
@@ -194,17 +211,6 @@ internal User(AbstractClient client)
 			this.Attributes = new Dictionary<string, JToken>();
 		}
 
-		public bool IsActive()
```

Comment:
@eugeninnowise We should have a deprecation notice here first, rather than removing this method first, since it is public. We can remove it in a few releases after deprecation.

Reply:
@vinaygahlawat I agree with you. But it is not possible to implement since member names (field, property, and method) must be unique.
https://stackoverflow.com/questions/15522242/can-a-property-name-and-a-method-name-be-same-in-c

Reply:
In this case then, let's name the property `Active`, so that we can preserve the `IsActive()` method and put an `[Obsolete]` attribute on it.

Reply:
agreed, deprecate for a few versions and remove later so you give the opportunity for users to adapt before make a breaking change